tools/dotcms-cli/cli/src/test/java/com/dotcms/api/client/files/PushServiceIT.java

```diff
@@ -96,20 +109,33 @@ void Test_Nothing_To_Push() throws IOException {
 
             final var folderPath = String.format("//%s", testSiteName);
 
-            var result = remoteTraversalService.traverseRemoteFolder(
-                    folderPath,
-                    null,
-                    true,
-                    new HashSet<>(),
-                    new HashSet<>(),
-                    new HashSet<>(),
-                    new HashSet<>()
-            );
-
             // Pulling the content
             OutputOptionMixin outputOptions = new MockOutputOptionMixin();
-            pullService.pullTree(outputOptions, result.getRight(), workspace.files().toAbsolutePath().toFile(),
-                    true, true, true, 0);
+
+            Map<String, Object> customOptions = Map.of(
+                    INCLUDE_FOLDER_PATTERNS, new HashSet<>(),
+                    INCLUDE_ASSET_PATTERNS, new HashSet<>(),
+                    EXCLUDE_FOLDER_PATTERNS, new HashSet<>(),
+                    EXCLUDE_ASSET_PATTERNS, new HashSet<>(),
+                    NON_RECURSIVE, false,
+                    PRESERVE, false,
+                    INCLUDE_EMPTY_FOLDERS, true
+            );
+
+            // Execute the pull
+            pullService.pull(
```

Comment:
I would change this to only take the PullOptions any additional param could make it into the PullOptions

Reply:
If that's OK, I will leave the `PullOptions` as the pull commands options holder but not as holder for the method parameters, but following your advice I added inside the `PullOptions` the `customOptions`.